appmetrics/app_metrics.go

```diff
@@ -211,7 +211,7 @@ func (am *AppMetrics) getAppData(guid string) (*App, error) {
 	return app, nil
 }
 
-func (am *AppMetrics) ParseAppMetric(envelope *events.Envelope) ([]metrics.MetricPackage, error) {
+func (am *AppMetrics) ParseAppMetric(envelope *events.Envelope, customTags []string) ([]metrics.MetricPackage, error) {
```

Comment:
customTags should be a part of the AppMetrics object on creation rather than passed into the parseContainerMetric on every single invocation. I think that'll be a bit more efficient

Reply:
I wish it were easier to add these further down in the pipeline, but I think that would require too big a change of the metrics types -- it should be done but let's hold off on it for now. We'll do it when we have to pay down some tech debt next time.